spec/currency_spec.rb

```diff
@@ -321,6 +321,16 @@ def to_s
       end
     end
 
+    describe "#iso?" do
+      it "Iso code true" do
+        expect(Money::Currency.new(:eur).iso?).to be true
+      end
+
+      it "Iso code false" do
```

Comment:
just FYI. It's better to use more fluent descriptions, which are read like usual English. Something like `it 'returns true for iso currency'`. You can find more examples and advices in http://betterspecs.org/